// @cloudscape-design/components/internal/analytics/context/analytics-context.js
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
import { createContext } from 'react';
/* istanbul ignore next */
export const FunnelContext = createContext({
    funnelInteractionId: undefined,
    setFunnelInteractionId: () => { },
    funnelType: 'single-page',
    optionalStepNumbers: [],
    totalFunnelSteps: 0,
    funnelSubmit: () => { },
    funnelCancel: () => { },
    submissionAttempt: 0,
    funnelNextOrSubmitAttempt: () => { },
    funnelState: { current: 'default' },
    errorCount: { current: 0 },
    loadingButtonCount: { current: 0 },
});
export const FunnelStepContext = createContext({
    funnelInteractionId: undefined,
    stepNameSelector: '',
    stepNumber: 0,
    funnelStepProps: {},
});
export const FunnelSubStepContext = createContext({
    funnelInteractionId: undefined,
    subStepId: '',
    stepNumber: 0,
    stepNameSelector: '',
    subStepSelector: '',
    subStepNameSelector: '',
    funnelStepProps: {},
});
