// @cloudscape-design/design-tokens/index.js
export var colorChartsRed300 = "var(--color-charts-red-300-23c2oa, #ea7158)";
export var colorChartsRed400 = "var(--color-charts-red-400-539ac2, #dc5032)";
export var colorChartsRed500 = "var(--color-charts-red-500-llkmm5, #d13313)";
export var colorChartsRed600 = "var(--color-charts-red-600-5wdull, #ba2e0f)";
export var colorChartsRed700 = "var(--color-charts-red-700-a6apbj, #a82a0c)";
export var colorChartsRed800 = "var(--color-charts-red-800-yx4mar, #972709)";
export var colorChartsRed900 = "var(--color-charts-red-900-rpnpq8, #892407)";
export var colorChartsRed1000 = "var(--color-charts-red-1000-qk5pc5, #7d2105)";
export var colorChartsRed1100 = "var(--color-charts-red-1100-pt1j9r, #721e03)";
export var colorChartsRed1200 = "var(--color-charts-red-1200-d626tx, #671c00)";
export var colorChartsOrange300 = "var(--color-charts-orange-300-885arv, #e07941)";
export var colorChartsOrange400 = "var(--color-charts-orange-400-8kv4m2, #cc5f21)";
export var colorChartsOrange500 = "var(--color-charts-orange-500-z22zi7, #bc4d01)";
export var colorChartsOrange600 = "var(--color-charts-orange-600-4myt4p, #a84401)";
export var colorChartsOrange700 = "var(--color-charts-orange-700-gaxkk6, #983c02)";
export var colorChartsOrange800 = "var(--color-charts-orange-800-io45li, #8a3603)";
export var colorChartsOrange900 = "var(--color-charts-orange-900-rz3zc9, #7e3103)";
export var colorChartsOrange1000 = "var(--color-charts-orange-1000-0qpjgf, #732c02)";
export var colorChartsOrange1100 = "var(--color-charts-orange-1100-grw837, #692801)";
export var colorChartsOrange1200 = "var(--color-charts-orange-1200-5nnge9, #602400)";
export var colorChartsYellow300 = "var(--color-charts-yellow-300-6c0xtx, #b2911c)";
export var colorChartsYellow400 = "var(--color-charts-yellow-400-u252ao, #9c7b0b)";
export var colorChartsYellow500 = "var(--color-charts-yellow-500-v2phgx, #8a6b05)";
export var colorChartsYellow600 = "var(--color-charts-yellow-600-4pv3hy, #7b5f04)";
export var colorChartsYellow700 = "var(--color-charts-yellow-700-3emu4t, #6f5504)";
export var colorChartsYellow800 = "var(--color-charts-yellow-800-edh0l0, #654d03)";
export var colorChartsYellow900 = "var(--color-charts-yellow-900-1zoghi, #5d4503)";
export var colorChartsYellow1000 = "var(--color-charts-yellow-1000-4ypqyp, #553f03)";
export var colorChartsYellow1100 = "var(--color-charts-yellow-1100-2bbrrm, #4d3901)";
export var colorChartsYellow1200 = "var(--color-charts-yellow-1200-pml6h8, #483300)";
export var colorChartsGreen300 = "var(--color-charts-green-300-xdy00p, #67a353)";
export var colorChartsGreen400 = "var(--color-charts-green-400-pyju52, #41902c)";
export var colorChartsGreen500 = "var(--color-charts-green-500-x1jom3, #1f8104)";
export var colorChartsGreen600 = "var(--color-charts-green-600-ax7yum, #1a7302)";
export var colorChartsGreen700 = "var(--color-charts-green-700-x1ubuk, #176702)";
export var colorChartsGreen800 = "var(--color-charts-green-800-erepp2, #145d02)";
export var colorChartsGreen900 = "var(--color-charts-green-900-jgvtf4, #125502)";
export var colorChartsGreen1000 = "var(--color-charts-green-1000-ftsnde, #104d01)";
export var colorChartsGreen1100 = "var(--color-charts-green-1100-8is059, #0f4601)";
export var colorChartsGreen1200 = "var(--color-charts-green-1200-pyehtq, #0d4000)";
export var colorChartsTeal300 = "var(--color-charts-teal-300-ze50a3, #2ea597)";
export var colorChartsTeal400 = "var(--color-charts-teal-400-eho3qj, #1c8e81)";
export var colorChartsTeal500 = "var(--color-charts-teal-500-ysfodg, #0d7d70)";
export var colorChartsTeal600 = "var(--color-charts-teal-600-k0lr0d, #096f64)";
export var colorChartsTeal700 = "var(--color-charts-teal-700-pradbd, #06645a)";
export var colorChartsTeal800 = "var(--color-charts-teal-800-l98pd3, #045b52)";
export var colorChartsTeal900 = "var(--color-charts-teal-900-2n2uqa, #03524a)";
export var colorChartsTeal1000 = "var(--color-charts-teal-1000-sqzv15, #014b44)";
export var colorChartsTeal1100 = "var(--color-charts-teal-1100-ugmadm, #01443e)";
export var colorChartsTeal1200 = "var(--color-charts-teal-1200-u1l5vv, #003e38)";
export var colorChartsBlue1300 = "var(--color-charts-blue-1-300-bhq5nc, #529ccb)";
export var colorChartsBlue1400 = "var(--color-charts-blue-1-400-6shq6z, #3184c2)";
export var colorChartsBlue1500 = "var(--color-charts-blue-1-500-u9fbb4, #0273bb)";
export var colorChartsBlue1600 = "var(--color-charts-blue-1-600-6cm0lb, #0166ab)";
export var colorChartsBlue1700 = "var(--color-charts-blue-1-700-pur7e5, #015b9d)";
export var colorChartsBlue1800 = "var(--color-charts-blue-1-800-d90xfc, #015292)";
export var colorChartsBlue1900 = "var(--color-charts-blue-1-900-ugbzx8, #014a87)";
export var colorChartsBlue11000 = "var(--color-charts-blue-1-1000-wn8i30, #01437d)";
export var colorChartsBlue11100 = "var(--color-charts-blue-1-1100-zh7t8t, #003c75)";
export var colorChartsBlue11200 = "var(--color-charts-blue-1-1200-7spkcj, #00366d)";
export var colorChartsBlue2300 = "var(--color-charts-blue-2-300-25paew, #688ae8)";
export var colorChartsBlue2400 = "var(--color-charts-blue-2-400-ppuj2w, #5978e3)";
export var colorChartsBlue2500 = "var(--color-charts-blue-2-500-30q04l, #4066df)";
export var colorChartsBlue2600 = "var(--color-charts-blue-2-600-bqdh5s, #3759ce)";
export var colorChartsBlue2700 = "var(--color-charts-blue-2-700-useohu, #314fbf)";
export var colorChartsBlue2800 = "var(--color-charts-blue-2-800-vbehwc, #2c46b1)";
export var colorChartsBlue2900 = "var(--color-charts-blue-2-900-rjxjr8, #273ea5)";
export var colorChartsBlue21000 = "var(--color-charts-blue-2-1000-fo6yfn, #23379b)";
export var colorChartsBlue21100 = "var(--color-charts-blue-2-1100-r56l0z, #1f3191)";
export var colorChartsBlue21200 = "var(--color-charts-blue-2-1200-mrmzh5, #1b2b88)";
export var colorChartsPurple300 = "var(--color-charts-purple-300-fks7nd, #a783e1)";
export var colorChartsPurple400 = "var(--color-charts-purple-400-g5p5yw, #9469d6)";
export var colorChartsPurple500 = "var(--color-charts-purple-500-teqjg9, #8456ce)";
export var colorChartsPurple600 = "var(--color-charts-purple-600-nf5r9n, #7749bf)";
export var colorChartsPurple700 = "var(--color-charts-purple-700-15kpqx, #6b40b2)";
export var colorChartsPurple800 = "var(--color-charts-purple-800-fskedv, #6237a7)";
export var colorChartsPurple900 = "var(--color-charts-purple-900-9ytuxe, #59309d)";
export var colorChartsPurple1000 = "var(--color-charts-purple-1000-yxj5hz, #512994)";
export var colorChartsPurple1100 = "var(--color-charts-purple-1100-meod30, #4a238b)";
export var colorChartsPurple1200 = "var(--color-charts-purple-1200-d8mpzv, #431d84)";
export var colorChartsPink300 = "var(--color-charts-pink-300-9pujzo, #da7596)";
export var colorChartsPink400 = "var(--color-charts-pink-400-s9k4qh, #ce567c)";
export var colorChartsPink500 = "var(--color-charts-pink-500-pic0fb, #c33d69)";
export var colorChartsPink600 = "var(--color-charts-pink-600-is3nog, #b1325c)";
export var colorChartsPink700 = "var(--color-charts-pink-700-wmp2jn, #a32952)";
export var colorChartsPink800 = "var(--color-charts-pink-800-u9xp2q, #962249)";
export var colorChartsPink900 = "var(--color-charts-pink-900-171x5m, #8b1b42)";
export var colorChartsPink1000 = "var(--color-charts-pink-1000-cm89wx, #81143b)";
export var colorChartsPink1100 = "var(--color-charts-pink-1100-0bdx0v, #780d35)";
export var colorChartsPink1200 = "var(--color-charts-pink-1200-af67f8, #6f062f)";
export var colorChartsStatusCritical = "var(--color-charts-status-critical-81ggtj, #7d2105)";
export var colorChartsStatusHigh = "var(--color-charts-status-high-qe7s3z, #ba2e0f)";
export var colorChartsStatusMedium = "var(--color-charts-status-medium-a45eb3, #cc5f21)";
export var colorChartsStatusLow = "var(--color-charts-status-low-vohc9l, #b2911c)";
export var colorChartsStatusPositive = "var(--color-charts-status-positive-bswwz8, #67a353)";
export var colorChartsStatusInfo = "var(--color-charts-status-info-97ztp9, #3184c2)";
export var colorChartsStatusNeutral = "var(--color-charts-status-neutral-3wprxa, #7d8998)";
export var colorChartsThresholdNegative = "var(--color-charts-threshold-negative-ax1vuv, #d91515)";
export var colorChartsThresholdPositive = "var(--color-charts-threshold-positive-5u7rd6, #037f0c)";
export var colorChartsThresholdInfo = "var(--color-charts-threshold-info-248fl2, #0972d3)";
export var colorChartsThresholdNeutral = "var(--color-charts-threshold-neutral-fe7vxm, #5f6b7a)";
export var colorChartsLineGrid = "var(--color-charts-line-grid-mz4lye, #d1d5db)";
export var colorChartsLineTick = "var(--color-charts-line-tick-arwxlx, #d1d5db)";
export var colorChartsLineAxis = "var(--color-charts-line-axis-j3qg2k, #d1d5db)";
export var colorChartsPaletteCategorical1 = "var(--color-charts-palette-categorical-1-s02edj, #688ae8)";
export var colorChartsPaletteCategorical2 = "var(--color-charts-palette-categorical-2-hd7uud, #c33d69)";
export var colorChartsPaletteCategorical3 = "var(--color-charts-palette-categorical-3-tm6hrf, #2ea597)";
export var colorChartsPaletteCategorical4 = "var(--color-charts-palette-categorical-4-fqnj32, #8456ce)";
export var colorChartsPaletteCategorical5 = "var(--color-charts-palette-categorical-5-h9jmtt, #e07941)";
export var colorChartsPaletteCategorical6 = "var(--color-charts-palette-categorical-6-z7e3fr, #3759ce)";
export var colorChartsPaletteCategorical7 = "var(--color-charts-palette-categorical-7-4w1lrz, #962249)";
export var colorChartsPaletteCategorical8 = "var(--color-charts-palette-categorical-8-xesix1, #096f64)";
export var colorChartsPaletteCategorical9 = "var(--color-charts-palette-categorical-9-93ht36, #6237a7)";
export var colorChartsPaletteCategorical10 = "var(--color-charts-palette-categorical-10-wus8nq, #a84401)";
export var colorChartsPaletteCategorical11 = "var(--color-charts-palette-categorical-11-uh7roa, #273ea5)";
export var colorChartsPaletteCategorical12 = "var(--color-charts-palette-categorical-12-ywqdlz, #780d35)";
export var colorChartsPaletteCategorical13 = "var(--color-charts-palette-categorical-13-0aws3q, #03524a)";
export var colorChartsPaletteCategorical14 = "var(--color-charts-palette-categorical-14-5l0pt5, #4a238b)";
export var colorChartsPaletteCategorical15 = "var(--color-charts-palette-categorical-15-glpjzo, #7e3103)";
export var colorChartsPaletteCategorical16 = "var(--color-charts-palette-categorical-16-73ijrp, #1b2b88)";
export var colorChartsPaletteCategorical17 = "var(--color-charts-palette-categorical-17-ltabw6, #ce567c)";
export var colorChartsPaletteCategorical18 = "var(--color-charts-palette-categorical-18-fog2ah, #003e38)";
export var colorChartsPaletteCategorical19 = "var(--color-charts-palette-categorical-19-ig4cn5, #9469d6)";
export var colorChartsPaletteCategorical20 = "var(--color-charts-palette-categorical-20-ta856w, #602400)";
export var colorChartsPaletteCategorical21 = "var(--color-charts-palette-categorical-21-ta0h08, #4066df)";
export var colorChartsPaletteCategorical22 = "var(--color-charts-palette-categorical-22-lj462v, #a32952)";
export var colorChartsPaletteCategorical23 = "var(--color-charts-palette-categorical-23-35jyqu, #0d7d70)";
export var colorChartsPaletteCategorical24 = "var(--color-charts-palette-categorical-24-eem6bw, #6b40b2)";
export var colorChartsPaletteCategorical25 = "var(--color-charts-palette-categorical-25-w302xt, #bc4d01)";
export var colorChartsPaletteCategorical26 = "var(--color-charts-palette-categorical-26-6ughlw, #2c46b1)";
export var colorChartsPaletteCategorical27 = "var(--color-charts-palette-categorical-27-57l9xa, #81143b)";
export var colorChartsPaletteCategorical28 = "var(--color-charts-palette-categorical-28-4q0mir, #045b52)";
export var colorChartsPaletteCategorical29 = "var(--color-charts-palette-categorical-29-cnn7dg, #512994)";
export var colorChartsPaletteCategorical30 = "var(--color-charts-palette-categorical-30-eix60t, #8a3603)";
export var colorChartsPaletteCategorical31 = "var(--color-charts-palette-categorical-31-jthnv1, #1f3191)";
export var colorChartsPaletteCategorical32 = "var(--color-charts-palette-categorical-32-h40wuu, #da7596)";
export var colorChartsPaletteCategorical33 = "var(--color-charts-palette-categorical-33-1iyt0i, #01443e)";
export var colorChartsPaletteCategorical34 = "var(--color-charts-palette-categorical-34-eut0yx, #a783e1)";
export var colorChartsPaletteCategorical35 = "var(--color-charts-palette-categorical-35-utj4hw, #692801)";
export var colorChartsPaletteCategorical36 = "var(--color-charts-palette-categorical-36-mjb3b0, #5978e3)";
export var colorChartsPaletteCategorical37 = "var(--color-charts-palette-categorical-37-6rz96d, #b1325c)";
export var colorChartsPaletteCategorical38 = "var(--color-charts-palette-categorical-38-s8j0ph, #1c8e81)";
export var colorChartsPaletteCategorical39 = "var(--color-charts-palette-categorical-39-5l5hgh, #7749bf)";
export var colorChartsPaletteCategorical40 = "var(--color-charts-palette-categorical-40-2i7lp3, #cc5f21)";
export var colorChartsPaletteCategorical41 = "var(--color-charts-palette-categorical-41-lozcy4, #314fbf)";
export var colorChartsPaletteCategorical42 = "var(--color-charts-palette-categorical-42-qzrtzy, #8b1b42)";
export var colorChartsPaletteCategorical43 = "var(--color-charts-palette-categorical-43-sq9ksf, #06645a)";
export var colorChartsPaletteCategorical44 = "var(--color-charts-palette-categorical-44-2mlngi, #59309d)";
export var colorChartsPaletteCategorical45 = "var(--color-charts-palette-categorical-45-irkhaj, #983c02)";
export var colorChartsPaletteCategorical46 = "var(--color-charts-palette-categorical-46-r0hukp, #23379b)";
export var colorChartsPaletteCategorical47 = "var(--color-charts-palette-categorical-47-l2wh9o, #6f062f)";
export var colorChartsPaletteCategorical48 = "var(--color-charts-palette-categorical-48-k8gno4, #014b44)";
export var colorChartsPaletteCategorical49 = "var(--color-charts-palette-categorical-49-e507mj, #431d84)";
export var colorChartsPaletteCategorical50 = "var(--color-charts-palette-categorical-50-hfws7f, #732c02)";
export var colorBackgroundButtonNormalActive = "var(--color-background-button-normal-active-dbl4gr, #d3e7f9)";
export var colorBackgroundButtonNormalDefault = "var(--color-background-button-normal-default-9p30rf, #ffffff)";
export var colorBackgroundButtonNormalDisabled = "var(--color-background-button-normal-disabled-avznb8, #ffffff)";
export var colorBackgroundButtonNormalHover = "var(--color-background-button-normal-hover-xiany8, #f2f8fd)";
export var colorBackgroundButtonPrimaryActive = "var(--color-background-button-primary-active-dubfz3, #033160)";
export var colorBackgroundButtonPrimaryDefault = "var(--color-background-button-primary-default-lxnvg2, #0972d3)";
export var colorBackgroundButtonPrimaryDisabled = "var(--color-background-button-primary-disabled-mp3gf0, #e9ebed)";
export var colorBackgroundButtonPrimaryHover = "var(--color-background-button-primary-hover-soejyt, #033160)";
export var colorBackgroundCellShaded = "var(--color-background-cell-shaded-oyxkf4, #f8f8f8)";
export var colorBackgroundContainerContent = "var(--color-background-container-content-og3y0v, #ffffff)";
export var colorBackgroundContainerHeader = "var(--color-background-container-header-l5cy7j, #ffffff)";
export var colorBackgroundControlChecked = "var(--color-background-control-checked-k2dd8u, #0972d3)";
export var colorBackgroundControlDefault = "var(--color-background-control-default-19p6ht, #ffffff)";
export var colorBackgroundControlDisabled = "var(--color-background-control-disabled-u2n10f, #d1d5db)";
export var colorBackgroundDropdownItemDefault = "var(--color-background-dropdown-item-default-8l3k44, #ffffff)";
export var colorBackgroundDropdownItemFilterMatch = "var(--color-background-dropdown-item-filter-match-zl9g0m, #f2f8fd)";
export var colorBackgroundDropdownItemHover = "var(--color-background-dropdown-item-hover-gvdk4u, #f4f4f4)";
export var colorBackgroundHomeHeader = "var(--color-background-home-header-qnhw1k, #000716)";
export var colorBackgroundInputDefault = "var(--color-background-input-default-y2omd4, #ffffff)";
export var colorBackgroundInputDisabled = "var(--color-background-input-disabled-nvrgh4, #e9ebed)";
export var colorBackgroundItemSelected = "var(--color-background-item-selected-ndi5zy, #f2f8fd)";
export var colorBackgroundLayoutMain = "var(--color-background-layout-main-m49752, #ffffff)";
export var colorBackgroundLayoutToggleActive = "var(--color-background-layout-toggle-active-csmprm, #354150)";
export var colorBackgroundLayoutToggleDefault = "var(--color-background-layout-toggle-default-q41mj7, #192534)";
export var colorBackgroundLayoutToggleHover = "var(--color-background-layout-toggle-hover-im3a7d, #354150)";
export var colorBackgroundLayoutToggleSelectedActive = "var(--color-background-layout-toggle-selected-active-kajyo0, #0972d3)";
export var colorBackgroundLayoutToggleSelectedDefault = "var(--color-background-layout-toggle-selected-default-ir617m, #0972d3)";
export var colorBackgroundLayoutToggleSelectedHover = "var(--color-background-layout-toggle-selected-hover-0qwzgy, #065299)";
export var colorBackgroundNotificationBlue = "var(--color-background-notification-blue-xqhf8m, #0972d3)";
export var colorBackgroundNotificationGreen = "var(--color-background-notification-green-dvafxg, #037f0c)";
export var colorBackgroundNotificationRed = "var(--color-background-notification-red-sul351, #d91515)";
export var colorBackgroundPopover = "var(--color-background-popover-qliu7h, #ffffff)";
export var colorBackgroundStatusError = "var(--color-background-status-error-9e19yh, #fff7f7)";
export var colorBackgroundStatusInfo = "var(--color-background-status-info-wlp7jc, #f2f8fd)";
export var colorBackgroundStatusSuccess = "var(--color-background-status-success-irojq1, #f2fcf3)";
export var colorBackgroundStatusWarning = "var(--color-background-status-warning-b15ytv, #ffffff)";
export var colorBackgroundToggleCheckedDisabled = "var(--color-background-toggle-checked-disabled-6e4fll, #b5d6f4)";
export var colorBorderButtonNormalDefault = "var(--color-border-button-normal-default-fctsfs, #0972d3)";
export var colorBorderButtonNormalDisabled = "var(--color-border-button-normal-disabled-kap33c, #9ba7b6)";
export var colorBorderButtonNormalHover = "var(--color-border-button-normal-hover-6kybh9, #033160)";
export var colorBorderButtonPrimaryDisabled = "var(--color-border-button-primary-disabled-30jhtd, #e9ebed)";
export var colorBorderContainerTop = "var(--color-border-container-top-camjrq, transparent)";
export var colorBorderControlDefault = "var(--color-border-control-default-v28icv, #7d8998)";
export var colorBorderDividerDefault = "var(--color-border-divider-default-9o8zql, #e9ebed)";
export var colorBorderDropdownItemHover = "var(--color-border-dropdown-item-hover-me98vk, #7d8998)";
export var colorBorderInputDefault = "var(--color-border-input-default-zb6g42, #7d8998)";
export var colorBorderItemFocused = "var(--color-border-item-focused-4t19h5, #0972d3)";
export var colorBorderItemSelected = "var(--color-border-item-selected-xoo46x, #0972d3)";
export var colorBorderStatusError = "var(--color-border-status-error-vuw4yz, #d91515)";
export var colorBorderStatusInfo = "var(--color-border-status-info-lubl5f, #0972d3)";
export var colorBorderStatusSuccess = "var(--color-border-status-success-9l0sch, #037f0c)";
export var colorBorderStatusWarning = "var(--color-border-status-warning-118bcl, #7d8998)";
export var colorForegroundControlDefault = "var(--color-foreground-control-default-rsunm6, #ffffff)";
export var colorForegroundControlDisabled = "var(--color-foreground-control-disabled-f2499o, #ffffff)";
export var colorTextAccent = "var(--color-text-accent-qh8xwn, #0972d3)";
export var colorTextBodyDefault = "var(--color-text-body-default-su7kuo, #000716)";
export var colorTextBodySecondary = "var(--color-text-body-secondary-he5d85, #414d5c)";
export var colorTextBreadcrumbCurrent = "var(--color-text-breadcrumb-current-u5dd2z, #5f6b7a)";
export var colorTextBreadcrumbIcon = "var(--color-text-breadcrumb-icon-e6o6qx, #7d8998)";
export var colorTextButtonNormalActive = "var(--color-text-button-normal-active-avnl7f, #033160)";
export var colorTextButtonNormalDefault = "var(--color-text-button-normal-default-x2bilf, #0972d3)";
export var colorTextButtonNormalHover = "var(--color-text-button-normal-hover-deauoa, #033160)";
export var colorTextButtonPrimaryActive = "var(--color-text-button-primary-active-dl9joj, #ffffff)";
export var colorTextButtonPrimaryDefault = "var(--color-text-button-primary-default-0h0aga, #ffffff)";
export var colorTextButtonPrimaryHover = "var(--color-text-button-primary-hover-03rhdd, #ffffff)";
export var colorTextCounter = "var(--color-text-counter-aen1n3, #5f6b7a)";
export var colorTextDropdownItemFilterMatch = "var(--color-text-dropdown-item-filter-match-t483bg, #0972d3)";
export var colorTextDropdownItemHighlighted = "var(--color-text-dropdown-item-highlighted-vjs5c4, #000716)";
export var colorTextEmpty = "var(--color-text-empty-v5xueo, #5f6b7a)";
export var colorTextFormDefault = "var(--color-text-form-default-yctx5s, #000716)";
export var colorTextFormSecondary = "var(--color-text-form-secondary-88dttc, #5f6b7a)";
export var colorTextGroupLabel = "var(--color-text-group-label-qtmgp9, #414d5c)";
export var colorTextHeadingDefault = "var(--color-text-heading-default-18iw0k, #000716)";
export var colorTextHeadingSecondary = "var(--color-text-heading-secondary-x2nd6m, #414d5c)";
export var colorTextHomeHeaderDefault = "var(--color-text-home-header-default-4zkonk, #ffffff)";
export var colorTextHomeHeaderSecondary = "var(--color-text-home-header-secondary-fa8niq, #d1d5db)";
export var colorTextInputDisabled = "var(--color-text-input-disabled-dmotcr, #9ba7b6)";
export var colorTextInputPlaceholder = "var(--color-text-input-placeholder-3y98yw, #5f6b7a)";
export var colorTextInteractiveActive = "var(--color-text-interactive-active-atr0kj, #000716)";
export var colorTextInteractiveDefault = "var(--color-text-interactive-default-2sml2m, #414d5c)";
export var colorTextInteractiveDisabled = "var(--color-text-interactive-disabled-nss20z, #9ba7b6)";
export var colorTextInteractiveHover = "var(--color-text-interactive-hover-8bpozs, #000716)";
export var colorTextInteractiveInvertedDefault = "var(--color-text-interactive-inverted-default-gkmiss, #d1d5db)";
export var colorTextInteractiveInvertedHover = "var(--color-text-interactive-inverted-hover-yrarfb, #fbfbfb)";
export var colorTextLabel = "var(--color-text-label-szhr80, #000716)";
export var colorTextLayoutToggle = "var(--color-text-layout-toggle-76lxyf, #ffffff)";
export var colorTextLayoutToggleActive = "var(--color-text-layout-toggle-active-lzrbla, #ffffff)";
export var colorTextLayoutToggleHover = "var(--color-text-layout-toggle-hover-dhzmbj, #0972d3)";
export var colorTextLayoutToggleSelected = "var(--color-text-layout-toggle-selected-qzrnpy, #ffffff)";
export var colorTextLinkDefault = "var(--color-text-link-default-gthbx2, #0972d3)";
export var colorTextLinkHover = "var(--color-text-link-hover-h1b9eu, #033160)";
export var colorTextNotificationDefault = "var(--color-text-notification-default-4icnky, #fbfbfb)";
export var colorTextStatusError = "var(--color-text-status-error-p6dh5n, #d91515)";
export var colorTextStatusInactive = "var(--color-text-status-inactive-5laoxv, #5f6b7a)";
export var colorTextStatusInfo = "var(--color-text-status-info-cmlm4f, #0972d3)";
export var colorTextStatusSuccess = "var(--color-text-status-success-6id1zv, #037f0c)";
export var colorTextStatusWarning = "var(--color-text-status-warning-3n8qp8, #d91515)";
export var colorTextTopNavigationTitle = "var(--color-text-top-navigation-title-hdg6ea, #000716)";
export var colorBoardPlaceholderActive = "var(--color-board-placeholder-active-kxfgmn, #e9ebed)";
export var colorBoardPlaceholderHover = "var(--color-board-placeholder-hover-8tcom5, #d3e7f9)";
export var colorDragPlaceholderActive = "var(--color-drag-placeholder-active-htcu4g, #e9ebed)";
export var colorDragPlaceholderHover = "var(--color-drag-placeholder-hover-w22j3x, #d3e7f9)";
export var fontFamilyBase = "var(--font-family-base-wn913o, 'Open Sans', 'Helvetica Neue', Roboto, Arial, sans-serif)";
export var fontFamilyMonospace = "var(--font-family-monospace-froq9j, Monaco, Menlo, Consolas, 'Courier Prime', Courier, 'Courier New', monospace)";
export var borderRadiusAlert = "var(--border-radius-alert-i70li9, 12px)";
export var borderRadiusBadge = "var(--border-radius-badge-tm5gnf, 4px)";
export var borderRadiusButton = "var(--border-radius-button-5qmvu7, 20px)";
export var borderRadiusCalendarDayFocusRing = "var(--border-radius-calendar-day-focus-ring-p4dxcv, 3px)";
export var borderRadiusContainer = "var(--border-radius-container-pmtq2d, 16px)";
export var borderRadiusControlCircularFocusRing = "var(--border-radius-control-circular-focus-ring-lj7j2y, 4px)";
export var borderRadiusControlDefaultFocusRing = "var(--border-radius-control-default-focus-ring-muizvi, 4px)";
export var borderRadiusDropdown = "var(--border-radius-dropdown-re1k6b, 8px)";
export var borderRadiusFlashbar = "var(--border-radius-flashbar-b42oa4, 12px)";
export var borderRadiusItem = "var(--border-radius-item-gyl0qr, 8px)";
export var borderRadiusInput = "var(--border-radius-input-g1eef0, 8px)";
export var borderRadiusPopover = "var(--border-radius-popover-g5ufqd, 8px)";
export var borderRadiusTabsFocusRing = "var(--border-radius-tabs-focus-ring-bz76gq, 20px)";
export var borderRadiusTiles = "var(--border-radius-tiles-2bi961, 8px)";
export var borderRadiusToken = "var(--border-radius-token-qzjnwb, 8px)";
export var borderRadiusTutorialPanelItem = "var(--border-radius-tutorial-panel-item-frshlz, 8px)";
export var motionEasingResponsive = "var(--motion-easing-responsive-1dqfrx, cubic-bezier(0, 0, 0, 1))";
export var motionEasingSticky = "var(--motion-easing-sticky-c9pq0x, cubic-bezier(1, 0, 0.83, 1))";
export var motionEasingExpressive = "var(--motion-easing-expressive-jp82i2, cubic-bezier(0.84, 0, 0.16, 1))";
export var motionDurationResponsive = "var(--motion-duration-responsive-h732s7, 115ms)";
export var motionDurationExpressive = "var(--motion-duration-expressive-gae032, 165ms)";
export var motionDurationComplex = "var(--motion-duration-complex-l5ma2h, 250ms)";
export var motionKeyframesFadeIn = "var(--motion-keyframes-fade-in-2qjwil, awsui-fade-in-35003c)";
export var motionKeyframesFadeOut = "var(--motion-keyframes-fade-out-ddkkh3, awsui-fade-out-35003c)";
export var motionKeyframesStatusIconError = "var(--motion-keyframes-status-icon-error-zq2kdx, awsui-status-icon-error-35003c)";
export var motionKeyframesScalePopup = "var(--motion-keyframes-scale-popup-58r4zv, awsui-scale-popup-35003c)";
export var spaceContainerHorizontal = "var(--space-container-horizontal-jxdgil, 20px)";
export var spaceFieldHorizontal = "var(--space-field-horizontal-8qo3uw, 12px)";
export var spaceScaledXxxs = "var(--space-scaled-xxxs-jdgipl, 2px)";
export var spaceScaledXxs = "var(--space-scaled-xxs-t2t62i, 4px)";
export var spaceScaledXs = "var(--space-scaled-xs-wbfgrv, 8px)";
export var spaceScaledS = "var(--space-scaled-s-913kwi, 12px)";
export var spaceScaledM = "var(--space-scaled-m-gxhdpl, 16px)";
export var spaceScaledL = "var(--space-scaled-l-2rs0gk, 20px)";
export var spaceScaledXl = "var(--space-scaled-xl-jawob6, 24px)";
export var spaceScaledXxl = "var(--space-scaled-xxl-fzmynl, 32px)";
export var spaceScaledXxxl = "var(--space-scaled-xxxl-crpygs, 40px)";
export var spaceStaticXxxs = "var(--space-static-xxxs-b6tgp7, 2px)";
export var spaceStaticXxs = "var(--space-static-xxs-jqh73n, 4px)";
export var spaceStaticXs = "var(--space-static-xs-zgeph4, 8px)";
export var spaceStaticS = "var(--space-static-s-q0yjjb, 12px)";
export var spaceStaticM = "var(--space-static-m-1plzkw, 16px)";
export var spaceStaticL = "var(--space-static-l-hozrvc, 20px)";
export var spaceStaticXl = "var(--space-static-xl-nmb7ge, 24px)";
export var spaceStaticXxl = "var(--space-static-xxl-qhb0le, 32px)";
export var spaceStaticXxxl = "var(--space-static-xxxl-srpnkv, 40px)";
export var shadowContainerActive = "var(--shadow-container-active-x0fzbz, 0px 1px 1px 1px #e9ebed, 0px 6px 36px #0007161a)";
